LFA.qml

```diff
@@ -122,6 +122,17 @@ Item {
 
     ////////// TRANSAXLE GEAR VARIABLES //////////////////////////////////////
     property real gearpos: rpmtest.geardata
+    property string gearinfo:    if(gearpos===9)"P"
```

Comment:
I'd replace the multiple if's with this:

 property string gearinfo: switch (gearpos) {
        case 0: return "N";
        case 1: return "1";
        case 2: return "2";
        case 3: return "3";
        case 4: return "4";
        case 5: return "5";
        case 6: return "6";
        case 7: return "7";
        case 8: return "8";
        case 9: return "P";
        case 10: return "R";
        default: return ""; // 100 is the value that says do not display gear position
    }